### PR TITLE
[dhd] Add support for discovery MER#1955

### PR DESCRIPTION
**Pull request by Thaodan**

### CODE REVIEW

File: droid-hal-discovery.spec [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# These and other macros are documented in dhd/droid-hal-device.inc
+# Feel free to cleanup this file by removing comments, once you have memorised them ;)
+
+%define device discovery
+%define vendor sony
+
+%define rpm_device discovery
```
Comment (sledges) [inline droid-hal-discovery.spec]: If you could remove this line, update dhd submodule to test with this commit https://github.com/mer-hybris/droid-hal-device/commit/0549fd71a8c3d21a6f37ef83952d00c207cbdf76 that would be great!

File: droid-hal-discovery.spec [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+%define rpm_vendor qualcomm
+
+%define vendor_pretty Sony
+%define device_pretty Xperia XA2 Ultra
```
Comment (sledges) [inline droid-hal-discovery.spec]: *_pretty are not used in dhd, you can cleanup this .spec as per pioneer.spec

File: droid-hal-discovery.spec (hunk 1 above)
Comment (Thaodan) [reply droid-hal-discovery.spec]: I'll try later when mer obs is back online.

**Comment by mkosola** [comment]
Could you remove droid-hal-disovery.spec file from this MR?

And can you add moutpoints for discovery to https://github.com/mer-hybris/hybris-boot?

**Comment by Thaodan** [comment]
Should I just include the pioneer spec file?

**Comment by mkosola** [comment]
There are currently two spec files in this pull request: droid-hal-disovery.spec and droid-hal-discovery.spec. Please remove droid-hal-disovery.spec.

**Comment by mkosola** [comment]
Great, thank you!

**Pull request merged**